### PR TITLE
[PC-1604] Docs 2.0 + Content Fixes - Portenta related changes

### PR DESCRIPTION
**Pull request by mcmchris**
## What This PR Changes
- Going through all these tasks fixing them: https://arduino.atlassian.net/browse/PC-1521

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/carriers/portenta-hat-carrier/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-hat-carrier/tutorials/user-manual/content.md:1]: ok!

File: content/hardware/04.pro/carriers/portenta-max-carrier/essentials.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-max-carrier/essentials.md:1]: Ok

**Review by jcarolinares — APPROVED** [review]
Approved

thank you @mcmchris let's merge

**Comment by mcmchris** [comment]
Merged!

**Pull request merged**